helm/memcached-operator/build/Dockerfile

```diff
@@ -0,0 +1,4 @@
+FROM quay.io/water-hole/helm-operator
```

Comment:
Did we not change this org name?

Reply:
We haven't yet. We'll have to come back and update once [operator-sdk/#832](https://github.com/operator-framework/operator-sdk/pull/832) is merged.